tests/datasource/test_pandas_datasource.py

```diff
@@ -299,7 +299,7 @@ def test_pandas_source_read_csv(
     reason="pyarrow and fastparquet are not installed",
 )
 @mock_s3
-@pytest.mark.filesystem
+@pytest.mark.aws_creds
```

Comment:
This one is definitely being mocked.

Reply:
I did see that too, but it also definitely fails in user forks

![Screenshot 2023-08-10 at 3 35 49 PM](https://github.com/great-expectations/great_expectations/assets/25670697/2ff65a61-7d18-4568-b04b-1408dfc305a7)

Reply:
This is the run I'm referring to
https://github.com/great-expectations/great_expectations/actions/runs/5801749091/job/15726800912?pr=8382

Reply:
Is `moto` not being installed?